src/Controls/tests/DeviceTests/Elements/Entry/EntryTests.Android.cs

```diff
@@ -164,5 +164,22 @@ public async Task RotationConsistent()
 			var platformRotation = await InvokeOnMainThreadAsync(() => platformEntry.Rotation);
 			Assert.Equal(expected, platformRotation);
 		}
+
+		[Fact(DisplayName = "Android crash when Entry has more than 5000 characters")]
+		[Category(TestCategory.Entry)]
+		public async Task EntryWithLongTextAndIsPassword_DoesNotCrash()
```

Comment:
In addition to test the initialization, could you include another test updating the Text value? https://github.com/dotnet/maui/blob/26732725ec21bf76796264e1d0addd475a17cbe7/src/Controls/tests/DeviceTests/Elements/Entry/EntryTests.Android.cs#L81